ci-connie.yml

```diff
@@ -135,9 +137,12 @@ post-processors:
     tag: "{{ user `docker_tag` }}"
     only:
     - docker
-  - type: artifice
-    files:
-    - tmp/docker-meta/.dumped
```

Comment:
This use of `artifice` should stay and not be replaced by the `manifest` post-processor, as we're effectively telling the following `shell-local` post-processor to perform `bin/docker-push` with the metadata provided in the `tmp/docker-meta/.dumped` file.